components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -489,11 +478,11 @@ interface ConversationHandler {
   // the assistant.
   SubmitSuggestion(string suggestion);
 
-  // Get associated page information. If there is none then |site_info| will be
-  // |null|. If the reason for that is it's still being fetched, then
+  // Get associated page information. If there is none then |associated_content|
+  // will be |null|. If the reason for that is it's still being fetched, then
   // |is_fetching| will be |true|.
```

Comment:
not sure why we still have the comment about is_fetching since that variable doesn't exist - maybe we should remove it in this PR (and from the event, below)?